mmv1/templates/terraform/constants/memorystore_diff.go.tmpl

```diff
@@ -0,0 +1,50 @@
+func desiredPscAndEndpointCustomizeDiff(_, old, new string, d *schema.ResourceData) bool {
```

Comment:
Can you wrap the function as following to not generate this function to TGC? Thanks.

```
{{- if ne $.Compiler "terraformgoogleconversion-codegen" }}
func desiredPscAndEndpointCustomizeDiff(_, old, new string, d *schema.ResourceData) bool {
...
}
{{- end }}
```